tools/wpt/install.py

```diff
@@ -2,20 +2,57 @@
 import browser
 import sys
 
+
+latest_channels = {
+    'firefox': 'nightly',
+    'chrome': 'dev',
+    'servo': 'nightly'
+}
+
+channel_by_name = {
+    'stable': 'stable',
+    'release': 'stable',
+    'beta': 'beta',
+    'nightly': latest_channels,
+    'dev': latest_channels,
+    'preview': latest_channels,
+    'experimental': latest_channels,
```

Comment:
The terms "experimental", "release" and "preview" aren't used by either browser. If we're going to support both "beta" and "newer than beta," I'm afraid those may confuse folks. Even if the confusion is rare, it seems better to omit them since they are all aliases for more familiar terms.

Alternatively, we could document their meaning in the `--help` text.

Reply:
I suppose "preview" might be used for the future Safari support, perhaps?